tests/conftest.py

```diff
@@ -46,6 +46,9 @@ def database_url(test_db):
     ).scalar()
     assert count_landsat == count_landsat_centroid
 
+    test_db.run_sql_file(os.path.join(DATA_DIR, "canada.sql"))
```

Comment:
@bitner really weird but adding this to the database make the tests to fail on another table 🤷